config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml

```diff
@@ -300,7 +300,7 @@ periodics:
       # 3.1.x. ETCD doesn't downgrade minor versions. So for downgrades, we
       # pin this to the lower version. The long term fix is to change
       # downgrade/upgrade to not upgrade/downgrade etcd.
-      - --env=TEST_ETCD_VERSION=3.0.17
+      - --env=TEST_ETCD_VERSION=3.1.10
```

Comment:
do we want this to match the [currently recommended minimum version](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#prerequisites) (3.2.10)?

Reply:
Yes, 3.2.10 please